src/Drupal/Commands/sql/SanitizeUserFieldsCommands.php

```diff
@@ -52,9 +52,13 @@ public function sanitize($result, CommandData $commandData)
         $conn = $this->getDatabase();
         $field_definitions = $this->getEntityFieldManager()->getFieldDefinitions('user', 'user');
         $field_storage = $this->getEntityFieldManager()->getFieldStorageDefinitions('user');
+        /** @deprecated Use $options['allowlist-fields'] instead. */
```

Comment:
Does this need an indicator on when the legacy value will be removed?

Reply:
Usually we would remove in the next major version. There is no Drush 11 branch at the moment, and no convention (other than `@deprecated`) to explicitly flag something for removal in Drush 11.

Drush 11 is far off, and it we don't have the bandwidth to maintain a "future" branch right now. We could consider introducing a convention, but in theory we remove all `@deprecated` functions in each major version, so `@deprecated` fills that role at the moment.